### ISSUE TITLE
`man-n browserify` errors

### ISSUE THREAD

**Issue by yoshuawuyts** [man-n/man-n#1]
Not sure what's going wrong, running version `0.0.2` on `io.js@2.3.0`.

```
❯ man-n browserify
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix
readlink: illegal option -- f
usage: readlink [-n] [file ...]
usage: dirname path
/Users/yw/.nvm/versions/io.js/v2.3.0/bin/man-n: line 29: : No such file or directory
npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/yw/.nvm/versions/io.js/v2.3.0/bin/iojs" "/Users/yw/.nvm/versions/io
js/v2.3.0/bin/npm" "view" "browserify" "readme"
npm ERR! node v2.3.0
npm ERR! npm  v3.0.0
npm ERR! code EPIPE
npm ERR! errno EPIPE
npm ERR! syscall write

npm ERR! write EPIPE
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/yw/src/yw/promise-each/npm-debug.log
No manual entry for
```

**Comment by wooorm** [comment]
Darn bash issues!

**Comment by yoshuawuyts** [comment]
Fixed in `v0.0.3`.
